gen3-integration-tests/tests/test_indexing_page.py

```diff
@@ -0,0 +1,120 @@
+            1. Login with indexing_account user
+            2. Navigate to indexing page
+            3. Upload valid manifest and wait for manifest-indexing sower job to finish
+            4. Check if the indexd record exists
+            5. Check if the indexd record hashes exists
+        """
+        login_page = LoginPage()
+        indexing_page = IndexingPage()
+        indexd = Indexd()
+        # Go to login page and login with indexing_account user
+        login_page.go_to(page)
+        login_page.login(page, user="indexing_account")
+        # Go to indexing page and validate page is loaded
+        indexing_page.go_to(page)
+        # Upload the valid manifest via indexing page
+        indexing_page.upload_valid_indexing_manifest(page)
+        # Check if the sowerjob pod for manifest-indexing has completed
+        gat.check_pod(pytest.namespace, "manifest-indexing", "sowerjob")
+        # Get the indexd record and check if the hash value matches to the test_hash value
+        index_record = indexd.get_record(self.variables["test_guid"])
+        indexd_record_hash = index_record["hashes"]["md5"]
+        logger.info(indexd_record_hash)
+        assert (
+            indexd_record_hash == self.variables["test_hash"]
+        ), f"Expected MD5 hash {self.variables['test_hash']}, but got {indexd_record_hash}"
+
+    def test_indexing_download(self, page):
+        """
+        Scenario: Login and navigate to indexing page and download indexd manifest
+        Steps:
+            1. Login with indexing_account user
+            2. Navigate to indexing page
+            3. Click on Download and wait for manifest-indexing sower job to finish
+            4. Send the request to the download url received from step 3
+            5. Verify the downloaded data
+        """
+        login_page = LoginPage()
+        indexing_page = IndexingPage()
+        # Go to login page and login with indexing_account user
+        login_page.go_to(page)
+        login_page.login(page, user="indexing_account")
+        # Go to indexing page and validate page is loaded
+        indexing_page.go_to(page)
+        manifest_link = indexing_page.download_manifest(page)
+        logger.debug(f"Download Link : {manifest_link}")
+        gat.check_pod(pytest.namespace, "indexd-manifest", "sowerjob")
+        # sending request with manifest_link to get manifest data
+        manifest_link_resp = requests.get(manifest_link)
+        logger.debug(manifest_link_resp.text)
+        manifest_data = manifest_link_resp.text
+        assert (
+            self.variables["expected_result"] in manifest_data
+        ), "Expected result not found in downloaded manifest"
+
+    def test_indexing_upload_invalid_manifest(self, page):
+        """
+        Scenario:
+        Steps:
+            1. Login with indexing_account user
+            2. Navigate to indexing page
+            3. Upload invalid manifest and wait for manifest-indexing sower job to finish
```

Comment:
it would be nice to document here in this docstring, or in `upload_invalid_indexing_manifest()`, what makes this manifest invalid